index.html

```diff
@@ -5,10 +5,10 @@
 	<meta http-equiv="content-type" content="text/html; charset=utf-8" />
 	<meta name="keywords" content="cv, resume, hResume, github, community, free, fun, nice, useful" />
 	<meta name="description" content="GitHub resumes, generated by the community, for the community." />
-	<link rel="stylesheet" type="text/css" href="http://yui.yahooapis.com/2.7.0/build/reset-fonts-grids/reset-fonts-grids.css" media="all" /> 
+	<link rel="stylesheet" type="text/css" href="css/reset-fonts-grids.css" media="all" /> 
 	<link rel="stylesheet" type="text/css" href="css/resume.css" media="all" />
 	<link rel="stylesheet" type="text/css" href="css/print.css" media="print" />
-	<script type="text/javascript" src="http://code.jquery.com/jquery-1.8.1.min.js"></script>
+	<script type="text/javascript" src="https://code.jquery.com/jquery-1.8.1.min.js"></script>
 	<script type="text/javascript" src="js/mustache.js"></script>
```

Comment:
"Now that SSL is encouraged for everyone
<https://www.eff.org/encrypt-the-web-report> and doesn’t have performance
concerns <https://istlsfastyet.com/>, *this technique is now an
anti-pattern*. If the asset you need is available on SSL, then *always* use
the https:// asset."
http://www.paulirish.com/2010/the-protocol-relative-url/
On 16 Feb 2016 08:56, "zhangyaning" <notifications@github.com> wrote:

> In index.html
> <https://github.com/resume/resume.github.com/pull/188#discussion_r52965724>
> :
>
> >  	<link rel="stylesheet" type="text/css" href="css/resume.css" media="all" />
> >  	<link rel="stylesheet" type="text/css" href="css/print.css" media="print" />
> > -	<script type="text/javascript" src="http://code.jquery.com/jquery-1.8.1.min.js"></script>
> > +	<script type="text/javascript" src="https://code.jquery.com/jquery-1.8.1.min.js"></script>
> >  	<script type="text/javascript" src="js/mustache.js"></script>
>
> // may be better?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/resume/resume.github.com/pull/188/files#r52965724>.
>